src/modules/core/components/Fields/Select/Select.tsx

```diff
@@ -124,9 +124,6 @@ const Select = ({
   const close = useCallback(() => {
     setIsOpen(false);
     setSelectedOption(-1);
-    if (comboboxRef.current) {
-      comboboxRef.current.focus();
-    }
```

Comment:
Since you removed this piece of logic, the `ref` itself doesn't have a point anymore.

Please remove:
- the `useRef` call an import
- the `ref` prop

Reply:
didint remove useRef import as useRef is also used by wrapper for handling outside click. Removed comboboxRef